README.md

```diff
@@ -51,17 +51,43 @@ a double hashtag like so:
 
 Lines with only a single `#` will not appear as section in `task:help` and can be seen as plain comments.
 
-### Help command
+## Help command
 
 Running `./Taskfile help`, the `task:help` function is triggered. This task will list all available sections and tasks
 using the double `##` comments you've learned about above. Now it's clear how you can run any other task!
 
-## Credits
+# Auto-completion
+
+Autocompletion works when you use `zsh` and `oh-my-zsh`. Create the following file in your oh-my-zsh directory
+`~/.oh-my-zsh/completions/_task.zsh`:
+
+```shell
+#compdef task
+
+_task() {
+	local -a commands
+	local tasks=$(task comp_targets)
+
+	while IFS= read -r line; do
+		if [[ -n "$line" ]]; then
+			commands+=("$line")
+		fi
+	done <<< "$tasks"
+
+	_describe -t commands 'task commands' commands
```

Comment:
What is `-t`, is there a written out variant available?

Reply:
https://zsh.sourceforge.io/Doc/Release/Completion-System.html#Completion-System, search for `_describe`:

> With the -t option a tag can be specified. The default is ‘values’ or, if the -o option is given, ‘options’. 

There is no alternative for this flag.